### PR TITLE
refactor/775

### LINKED ISSUE #775: KSEA needs to be updated to AIRAC 1711
Main changes since previous version:
-All SID's have been renamed
-HAWKZ5 and MARNR6 Arrivals have been renamed

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #775

### CODE REVIEW

**Comment by erikquinn** [comment]
@Fechulo any chance you could re-do the terrain on this one? I'm growing tired of the `terrain below sea level is not supported!` warnings. 😉

**Comment by Fechulo** [comment]
@erikquinn will do!

**Comment by erikquinn** [comment]
@Fechulo thank you!!!

File: assets/airports/ksea.json [hunk 1]
```diff
@@ -384,9 +384,9 @@
                 ["YVR*"]
             ]
         },
-        "SEATL5": {
-            "icao": "SEATL5",
-            "name": "Seattle Five",
+        "SEA6": {
```
Comment (erikquinn) [inline assets/airports/ksea.json]: Thank god they changed this identifier, `SEATL5` was _super_ stupid and nonconformist. Bunch of hipsters... then again, _Seattle_... 🤔

**Pull request merged**